dependabot/build.gradle

```diff
@@ -7,12 +7,16 @@ dependencies {
     api "org.apache.ant:ant:${project['ant.version']}"
     api "org.apache.ant:ant-junit:${project['ant.version']}"
     api "org.asciidoctor:asciidoctorj:${project['asciidoctorj.version']}"
+    api "com.bertramlabs.plugins:asset-pipeline:${project['asset-pipeline.version']}"
```

Comment:
`com.bertramlabs.plugins:asset-pipeline` is not right. Did you mean `asset-pipeline-core`?
Also, these versions has historically been out of sync. For example, there is no `asset-pipline-grails:4.4.0`.

Reply:
ok, poor assumption on my part.